content/en/docs/platforms/kubernetes/operator/automatic.md

```diff
@@ -79,10 +79,9 @@ spec:
       batch:
         send_batch_size: 10000
         timeout: 10s
-
     exporters:
-      # NOTE: Prior to v0.86.0 use `logging` instead of `debug`.
```

Comment:
I think we might want to keep this comment, but I'll defer to the @open-telemetry/collector-approvers.